lib/elasticsearch.ts

```diff
@@ -86,7 +86,7 @@ function filter_old_indices(current_indices) {
     const acceptable_indices = [];
     let today = moment();
     for (let i = 0; i < config.indices.days; i++) {
-      acceptable_indices.push(`${config.indices.prefix}-${today.format("YYYY.MM.DD")}`);
+      acceptable_indices.push(`${config.indices.prefix}-${today.format("YYYY-MM-DD")}`);
```

Comment:
NIT: can we move this to a const so that we don't need to edit it in many places?